mlflow/pyfunc/__init__.py

```diff
@@ -141,14 +141,14 @@ def _get_code_dirs(src_code_path, dst_code_path=None):
             and not x == "__pycache__"]
 
 
-def spark_udf(spark, path, run_id=None, result_type="double"):
+def load_spark_udf(spark, path):
```

Comment:
Why does this not take run_id anymore?

Reply:
Oh, I meant to remove this one I think. I will double check but I thinkt it's a leftover from where I had spark_udf as meta_flavor and flavors supporting spark_udf mode would have load_spark_udf method.